Overworld.gd

```diff
@@ -28,6 +28,11 @@ func place_player():
 func _process(delta):
 	counter += delta
 	$CanvasLayer/Counter.text = "Counter: " + str(counter)
+	var cpos = $TileMap.world_to_map($Player.position)
```

Comment:
What does `world_to_map` do?

Reply:
takes a world position like (576, 256) and maps it to the tile based grid (18, 8)
formula being tile_pos = Vector2(pos.x / tile_size.x , pos.y /tile_size.y)
the built in method takes this into account and figures out the conversion